client/src/components/Share.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from "react";
+import { constructShareable } from "../utilities/shareable";
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { apiURI } from '../main';
+
+export const Share = (props) => {
+    const [results, setResults] = useState();
+    const [copied, setCopied] = useState(false);
+
+    const readyResults = async () => {
+        try {
+            await fetch(`${apiURI}/country/load`)
+                    .then((res) => res.json())
+                    .then((data) => {
+                        const numOfGuesses = props.stats.guesses + 1; // If this is done inline, it will sometimes return 0 for guesses
```

Comment:
```suggestion
                        const numOfGuesses = props.stats.guesses; // If this is done inline, it will sometimes return 0 for guesses
```

Reply:
guesses inneholder ikke det siste gjettet man gjør - det holder bare på land som man har gjettet, derfor +1 .

Reply:
glem det

Reply:
det jeg tenkte først var litt feil.

props.stats.guesses er en for liten tror jeg?

props.data.guesses er et array med alle gjett. også det landet du skal fram til når du tilslutt gjetter det.

```suggestion
                        const numOfGuesses = props.data.guesses; // If this is done inline, it will sometimes return 0 for guesses
```

vil ikke 'ledertavle' skjermen vise en for lite?

Reply:
`props.stats.guesses` er initialisert med `props.data.guesses.length`, ref. linje 34 i `VictoryBox.jsx`